sdk/containerregistry/azure-containerregistry/README.md

```diff
@@ -1,35 +1,105 @@
-<!-- [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=master) -->
+# Azure Container Registry client library for Python
 
-# Azure Template Package client library for Python
+Azure Container Registry allows you to store and manage container images and artifacts in a private registry for all types of container deployments.
 
-This template package matches necessary patterns that the development team has established to create a unified sdk functional from Python 2.7 onwards. The packages contained herein can be installed singly or as part of the `azure` namespace. Any other introductory text should go here.
+Use the client library for Azure Container Registry to:
 
-This package has been tested with Python 2.7, 3.5, 3.6, 3.7 and 3.8
+- List images or artifacts in a registry
+- Obtain metadata for images and artifacts, repositories and tags
+- Set read/write/delete properties on registry items
+- Delete images and artifacts, repositories and tags
 
-For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
+[Source code][source] | [Package (Pypi)]<!--[package]--> | [API reference documentation]<!--[docs]--> | [REST API documentation][rest_docs] | [Product documentation][product_docs]
```

Comment:
```suggestion
[Source code][source] | [Package (Pypi)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
```

Reply:
These links aren't functional yet and will have to be added in right after release.